cdmtaskservice/models.py

```diff
@@ -616,6 +615,13 @@ def get_files_per_container(self) -> list[list[S3File | str]]:
             files.append(infiles[:fcount])
             infiles = infiles[fcount:]
         return files
+    
+    def get_total_compute_time_sec(self) -> float:
+        """
+        Return the total compute time as seconds for the job, calculated as
+        cpus * containers * runtime.
```

Comment:
Is `runtime` a user input estimation? In practice, there are numerous factors that can influence the `runtime` making `get_total_compute_time_sec` never accurate.

Reply:
yes, there's no way to get it otherwise AFAIK - the service doesn't know anything about the containers it's running. And yes, I totally agree, but I don't know how to do better without a lot more work. This is just a check to make sure the user isn't doing something insane; it doesn't need to be super accurate.

One thing I've thought about doing was adding some sort of metadata to the image registration that could be used to estimate a runtime based on the file count and sizes, but doing anything like that would be far in the future.